packages/autocomplete/src/parse.ts

```diff
@@ -23,10 +24,18 @@ export function parseAutocomplete(template: string, theme: any = {}): ParsedAuto
           })
         }
         if (key in theme) {
-          parts.push({
-            type: 'group',
-            values: Object.keys(theme[key]),
-          })
+          if (attrs.includes('deep')) {
+            parts.push({
+              type: 'deepgroup',
```

Comment:
Don't we just enumerate the combinations add a group type so we don't need to have more logics to handle afterwards?

Reply:
idk since enumerating the combinations is sort of like doing a `listAll`.

Reply:
`text-` generates ~700 color suggestions and it's the only one nesting so I'm still not sure, you can merge it and copy the `resolve` function in `listAll` part to get the group.